components/badge/badge.tsx

```diff
@@ -3,18 +3,34 @@ import { badge } from "./style";
 
 export interface IBadgeProps {
   appearance?: "default" | "success" | "primary" | "warning" | "danger" | "outline";
+  onClick?: (event: React.MouseEvent<HTMLElement>) => void;
   children: JSX.Element | string;
 }
 
+export interface IOpts {
+  appearance: IBadgeProps["appearance"];
+  isEvent: boolean;
+}
 export class Badge extends React.PureComponent<IBadgeProps, {}> {
   public static defaultProps: Partial<IBadgeProps> = {
     appearance: "default"
   };
 
   public render() {
-    const { appearance, children } = this.props;
+    const { appearance, onClick, children } = this.props;
+    const isOnClickOn = !!onClick;
```

Comment:
Would you mind going with `Boolean(onClick)` here? I know it's more characters but it's also more intuitive IMHO

Reply:
👍